src/SelectSingle.vue

```diff
@@ -172,6 +170,7 @@
 				}
 			},
 			handleOptionClick(event: MouseEvent, index: number) {
+				this.selectOption(index)
```

Comment:
Click on option doesn't change `selectedIndex` and `inputValue` vars.
![ezgif com-gif-maker](https://user-images.githubusercontent.com/64218169/99802324-94432480-2b48-11eb-8130-341b030eca5f.gif)

Reply:
Hi @ee923925github - I'm realizing I don't understand how this is working right now 🤔 

How was it working *without* this line? What was causing the value to be updated? I guess that `this.$emit('select')` would be called in `updateMenuState`? And that would trigger the update of the `v-model`?

Reply:
I think that makes sense. Let's do it; thanks for investigating this!

Reply:
yes, it worked because of emitted event `select`. But component's variables were not updated.